DEVELOPMENT.md

```diff
@@ -19,47 +45,36 @@ To check out this repository:
 1. Create a [fork](https://help.github.com/en/articles/fork-a-repo) of this repo
 2. Create the directories and clone your fork
 
-```
-mkdir -p $GOPATH/src/github.com/KohlsTechnology
-cd $GOPATH/src/github.com/KohlsTechnology
-git clone https://github.com/<YOUR FORK>/eunomia.git
+```shell
+git clone https://github.com/<YOUR_FORK>/eunomia.git
 cd eunomia
 ```
 
-### Installing on a Mac
+## Running Locally for Development Purposes
+
+The most efficient way to develop the operator locally is run the code on your local machine. This allows you to test code changes as you make them.
 
-All the components can easily be installed via [Homebrew](https://brew.sh/) on a Mac:
+To achieve this, run the helper script (preferably in a separate console window as the script blocks):
 
 ```shell
-brew install git
-brew install go
-brew install docker
-brew install operator-sdk
-brew install minikube
-brew install minishift
+./scripts/run-eunomia-locally.sh
 ```
 
-## Running Locally for Development Purposes
+If you want Eunomia to watch a particular namespace, just pass its name as an argument to the above script. Running the script without the argument defaults to Eunomia watching all namespaces.
 
-The most efficient way to develop the operator locally is run the code on your local machine. This allows you to test code changes as you make them.
+### **Note on template-processor images**
 
-```
-minikube start
-kubectl apply -f ./deploy/crds/eunomia_v1alpha1_gitopsconfig_crd.yaml
-export JOB_TEMPLATE=./build/job-templates/job.yaml
-export CRONJOB_TEMPLATE=./build/job-templates/cronjob.yaml
-export WATCH_NAMESPACE=""
-export OPERATOR_NAME=eunomia-operator
-export GO111MODULE=on
-go mod vendor
-operator-sdk up local --namespace="${WATCH_NAMESPACE}"
+Eunomia binary being run locally still uses [template-processor images](./README.md#Template-Engine) to manage templates and parameters. If you are tweaking the images (anything in `template-processors` directory), make sure to deploy them to your local Minikube Docker registry beforehand. The easiest way is to use the helper script:
+
+```shell
+./scripts/deploy-to-local.sh minikube
```

Comment:
or minishift?

Reply:
In 9b07ac9feb5d95390d9e96b0f4520a55e4e9561a I added a note that running eunomia locally enforces the use of a Kubernetes cluster